src/content/docs/guides/hacktoberfest.md

```diff
@@ -0,0 +1,67 @@
+
+:::note
+Participating Astro repositories, rules, instructions, and rewards may vary from year to year. Please consult this guide for up-to-date information every October!
+:::
+
+## Join the fest
+
+To take part in Hacktoberfest, here's all you need to do:
+
+1. Register for the event on [Hacktoberfest's official website](https://hacktoberfest.com/).
+2. Make any [valid contributions](#allowed-contributions) following [Hacktoberfest's rules](https://hacktoberfest.com/participation/#pr-mr-details) during October.
+3. Let us know you're participating by requesting the PRs label `hacktoberfest-accepted`.
+
+## How to contribute to open-source
+
+Especially for first-time contributors, open-source can be quite frightening -- but don't worry, we're here to help you succeed! 
+
+In addition to [our own resources for first-time contributors](/first-time/1-about/), here are a few reliable resources to help you prepare for your first Hacktoberfest contributions:
+
+- [What is open-source](https://www.digitalocean.com/community/tutorials/what-is-open-source)
+- [How to contribute to open-source](https://opensource.guide/how-to-contribute/)
+- [How to use Git](https://www.digitalocean.com/community/cheatsheets/how-to-use-git-a-reference-guide)
+- [Getting started with GitHub Desktop](https://docs.github.com/en/desktop/overview/getting-started-with-github-desktop)
+
+## Allowed contributions
+
+Astro welcomes contributions of all sizes from contributors of any experience level to this year's [participating repositories](#participating-repositories). **To be considered for Hacktoberfest credit, the contribution must improve the quality of the repository with a helpful addition, improvement, or fix.**
+
+Small fixes such as correcting typos are welcome and appreciated! Larger contributions such as translating a page of documentation or submitting a fix to an existing issue may also count towards Hacktoberfest totals if the PR can be reviewed and merged during the month of October.
+
+We will be closing PRs that do not meet our quality standards. Spam submissions will not be tolerated, and may result in being banned from the repo.
+
+:::caution
+**AI contributions will not be accepted.** Hacktoberfest exists to motivate and reward human contributions to open-source projects, and therefore, we cannot accept contributions that have been made by automated tools.
+
+AI assistance tools, such as GitHub Copilot or Cursor, may be used to help you draft your contribution but must be checked and edited for accuracy. If your submission contains obviously AI-generated errors, your PR will be rejected and you may be banned from further Hacktoberfest participation at our discretion.
+:::
+
+## Participating repositories
+
+### Astro Docs
+
+The [`withastro/docs` repository](https://github.com/withastro/docs) contains the complete source code and content for the [Astro documentation](https://docs.astro.build/) website in all the available languages.
```

Comment:
```suggestion
[`withastro/docs`](https://github.com/withastro/docs) contains the complete source code and content for the [Astro documentation](https://docs.astro.build/) website in all the available languages.
```
This formatting looked a little ugly/overwhelming visually, so let's optimize for people scanning and looking for repos!